junit-plugin/src/main/scala/org/junit/RuntimeStubs.scala

```diff
@@ -0,0 +1,12 @@
+package org.junit
+
+/* Stubs
+ * Defined in `junit-runtime`
+ */
+class Test
```

Comment:
I added it when I removed the dependency on `jUnitRuntime`. Those dummies are used in the plugin code to generate the type objects (like `typeOf[org.junit.Test]`). Without the dummies it does no compile. 
Is there a better way to do this without depending on `jUnitRuntime`?